config/src/config/state_sync_config.rs

```diff
@@ -153,3 +156,19 @@ impl Default for DataStreamingServiceConfig {
         }
     }
 }
+
+#[derive(Copy, Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[serde(default)]
+pub struct DiemDataClientConfig {
+    pub timeout: Duration, // Timeout used by the network client when sending requests
```

Comment:
- nit maybe rename this `response_timeout_ms`? It's the timeout we have when waiting for a response. 
- Also, maybe rename `data_summary_poll_interval` to `summary_poll_interval_ms` just so it's clear what time unit we're using. We've had issues in the past where a mismatch in time units meant that things were not working correctly.

Reply:
Thanks!! These are important nits. I made the following changes:

- Renaming struct members to include `_ms`.
- Updating comments.